ccpp/data/GFS_typedefs.meta

```diff
@@ -716,6 +730,30 @@
   dimensions = (horizontal_loop_extent)
   type = real
   kind = kind_phys
+[dust12m_in]
+  standard_name = fengsha_dust12m_input
+  long_name = fengsha dust input
+  units = various
+  dimensions = (horizontal_dimension,12,5)
+  type = real
+  kind = kind_phys
+  active = (flag_for_rrfs_smoke_coupling)
+[emi_in]
+  standard_name = anthropogenic_background_input
+  long_name = anthropogenic background input
+  units = various
+  dimensions = (horizontal_dimension,1)
+  type = real
+  kind = kind_phys
+  active = (flag_for_rrfs_smoke_coupling)
+[smoke_GBBEPx]
+  standard_name = emission_smoke_GBBEPx
```

Comment:
How about `hourly_smoke_emission`

Reply:
No. That is specifically the GBBEPx instrument's measurements of smoke emissions.

Reply:
How are the GBBEPx model's smoke emissions estimates used? Could there conceivably be a different model's smoke emission estimate? Can this name be generalized to `hourly_modeled_smoke_emissions` or something?